src/sentry/api/endpoints/organization_event_details.py

```diff
@@ -41,7 +41,9 @@ def get(self, request, organization, project_slug, event_id):
             return Response({"detail": "Event not found"}, status=404)
 
         reference = None
-        fields = request.query_params.getlist("field")
+        fields = [
+            field for field in request.query_params.getlist("field") if not field.strip() == ""
```

Comment:
Does a similar change need to be made for the organization_events endpoint?

Reply:
`organization_events` handles this case with a check in `resolve_field_list`, but I didn't want to use it here since it would require building a Filter object, and would be more than what this endpoints needs.